test/distributed/test_functional_api.py

```diff
@@ -427,7 +427,7 @@ def allred_mesh_dim(input):
 
 
 def exit_if_lt_x_gpu(x):
-    if BACKEND == dist.Backend.NCCL and torch.cuda.device_count() < x:
+    if not torch.cuda.is_available() or torch.cuda.device_count() < x:
```

Comment:
if cuda is not available, then device count should be 0
```suggestion
    if torch.cuda.device_count() < x:
```